.metadata

```diff
@@ -0,0 +1,210 @@
+# Copyright 2019 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+# ----------------------------------------------------------------------
+---
+# command line arguments
+params:
+  existing_version: 4.3.0
+  new_version: 4.4.0
+  wso2sp: wso2sp-4.4.0
+# files based configurations
+files:
+  - file_path: pattern-distributed/sp/wso2sp-dashboard-deployment.yaml
```

Comment:
Shall we add comments on file_paths? IMHO, easier to read

Reply:
Could you please elaborate with an example